docs/fastdds/xml_configuration/common.rst

```diff
@@ -351,6 +352,61 @@ Please refer to :ref:`threadsettingsqos` for further information on ResourceLimi
      - ``int32_t``
      - -1
 
+.. _BuiltinTransportType:
+
+BuiltinTransports
+^^^^^^^^^^^^^^^^^
+
+It controls the builtin transports that will be used during the initialization of the DomainParticipant.
+Please refer to :ref:`rtps_layer_builtin_transports` for further information on builtin transports.
+
+This tag can be configured with the following attributes:
```

Comment:
I would include this section right before the next one, so we can introduce it as: 
```suggestion
The ``<builtinTransport>`` tag can be configured with the following attributes:
```